projects/online/online/utils/searcher.py

```diff
@@ -123,9 +123,11 @@ def __init__(
         self.last_detection_time = time.time() - self.refractory_period
 
         # calculate the detection statistic threshold
-        # corresponding to the requested FAR threshodl
-        self.background = background
-        self.threshold = self.background.threshold_at_far(far_threshold)
+        # corresponding to the requested FAR threshold
+        self.threshold = background.threshold_at_far(far_threshold)
+        # Speed up FAR calculation by excluding below-threshold events
```

Comment:
This won't affect the underlying FAR value since `num_events` or whatever is already fixed?

Reply:
Right - we're just counting the number of background events above the detection statistic of the candidate event, so background events below the detection threshold don't matter.